Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -505,7 +505,7 @@ void paint_text_decoration(PaintContext& context, Layout::Node const& text_node,
     auto& font = fragment.layout_node().first_available_font();
     auto fragment_box = fragment.absolute_rect();
     CSSPixels glyph_height = CSSPixels::nearest_value_for(font.pixel_size());
-    auto baseline = fragment_box.height() / 2 - (glyph_height + 4) / 2 + glyph_height;
+    auto baseline = fragment.baseline();
```

Comment:
What the..
I assume this affected visual results?

Reply:
I didn't manage to tell visual difference when I was testing this. it was like 14.98px after vs 15px before.

Reply:
also added a ref-test for text decorations.